app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt

```diff
@@ -112,19 +112,29 @@ class EmbarkActivity : BaseActivity(R.layout.activity_embark) {
         if (offerKeys != null && offerKeys.isNotEmpty()) {
             val offerIds = model.getListFromStore(offerKeys)
             startActivity(OfferActivity.newInstance(this, offerIds))
-        } else if (embarkData.passage?.name == "Offer") {
+            return
+        }
+        if (embarkData.passage?.name == "Offer") {
```

Comment:
Should we remove this now that we always want to show native offer screen?

Reply:
Good point. Yes!